src/components/MAPI/clusters/CreateCluster/patches.ts

```diff
@@ -65,6 +65,13 @@ export function withClusterReleaseVersion(
       controlPlaneNode.metadata.namespace = hasNonNamespacedResources
         ? defaultNamespace
         : orgNamespace;
+      if (
+        providerCluster &&
+        controlPlaneNode.kind === infrav1alpha3.G8sControlPlane
+      ) {
+        controlPlaneNode.spec.infrastructureRef.namespace =
```

Comment:
We set the namespace for the control plane node's infrastructure reference to be the same one as `AWSControlPlane` CR's namespace.